### DISCUSSION TITLE
Error When Trying to Sign Up

### DISCUSSION THREAD

**Discussion by SOMONSOUM** [highlight/highlight#9431] [Q&A] [answered]
I followed the [Hobby deployment guide](https://www.highlight.io/docs/getting-started/self-host/self-hosted-hobby-guide) to deploy Highlight on my VPS, and I received the following output:

`
✔ Container collector   Healthy                                                                                     1.9s
✔ Container zookeeper   Healthy                                                                                     1.9s
✔ Container kafka       Healthy                                                                                     1.9s
✔ Container redis       Healthy                                                                                     1.9s
✔ Container postgres    Healthy                                                                                     1.9s
✔ Container clickhouse  Healthy                                                                                     1.9s
Highlight infrastructure started
[+] Pulling 2/2
✔ backend Pulled                                                                                                    0.8s
✔ frontend Pulled                                                                                                   0.8s
`
<img width="489" alt="highlight" src="https://github.com/user-attachments/assets/6ed45411-3028-488b-b3f9-9a096ccf9699">

Highlight started on `http://localhost:3000`
However, when I try to access http://my-ip:3000 and create a new account, I encounter a `Failed to fetch error`. I’m not sure what is causing this issue. Could anyone help me resolve it?

**Comment by Vadman97** [comment, marked as answer]
@SOMONSOUM , did you update the `docker/.env` file with the IP address / hostname for your frontend / backend? You'll need to do that and rerun `./run-hobby.sh` to restart the docker containers.
